src/iac/client/webm_opus.h

```diff
@@ -0,0 +1,35 @@
+#ifndef WEBM_OPUS_H
+#define WEBM_OPUS_H
+
+#include <stdio.h>
+#include <stdint.h>
+#include <opus/opus.h>
+// #include <speex/speex_resampler.h> // Removed SpeexDSP include
+
+// WebM/Opus related constants
+#define OPUS_MAX_PACKET_SIZE 1500
+#define OPUS_SAMPLE_RATE 16000 // Expect 16kHz Opus output now
```

Comment:
The Opus decoder can be configured to decode to a specific sample rate. We don't need to use this constant and can rely upon the IAD configuration.